README.md

```diff
@@ -43,6 +43,7 @@ A curated list of amazingly awesome PHPStorm plugins, resources and other shiny
 ### Testing
 
 * [PHPUnit Autocomplete Assistant](https://plugins.jetbrains.com/plugin/7722-phpunit-autocomplete-assistant) - PhpStorm plugin to provide smart autocomplete, code navigation and refactoring features for mocked class methods.
+* [PHPUnit Enhancement](https://plugins.jetbrains.com/plugin/9674-phpunit-enhancement) - Fork of PHPUnit Autocomplete Assistant to support latest PhpStorm API and provide additional PhpUnit / Prophecy features
```

Comment:
Could you replace `PHPUnit Autocomplete Assistant` instead? Makes more sense to have the updated version on the last than the original that hasn't been updated in a year tbh 😄 .

Reply:
done